CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # UnlimitedGPT Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.1.5.6] 2023/06/11
+- Added `tests` folder for `pytest`.
```

Comment:
No need to add a whole version just for the tests. I usually release a new version only if there was a new feature or a bug was fixed.